src/test/java/plugins/AnalysisCollectorPluginTest.java

```diff
@@ -329,6 +330,33 @@ public void should_aggregate_warnings_in_dashboard_portlet() {
         assertThat(dashboard, not(hasWarningsFor(job, TASKS, TASKS_ALL)));
     }
 
+    @Test
+    @WithPlugins({"git@2.4.0", "workflow-aggregator@1.10"})
```

Comment:
would it make sense to add `@Issue` with a link to the issue you re trying to test?